ringbuffer.py

```diff
@@ -0,0 +1,32 @@
+import collections
+
+
+class RingBuffer(collections.deque):
```

Comment:
oic, you wrote your own ringbuffer.  Why not use https://pypi.org/project/ringbuf/?

Reply:
Well because I did not see this one.  If only I had known.